meteor-app/imports/api/orders/methods.js

```diff
@@ -20,6 +22,21 @@ Meteor.methods({
       throw new Meteor.Error(500, e.sanitizedError.reason)
     }
   },
+  async 'order.archive'(order) {
+    try {
+      log.info('moving order to archive ', order._id)
+      await Orders.insert ({
+        status: 1,
```

Comment:
Please use constants - we went to the bother of defining them, and out of context I have no idea what a status value of '1' means, but if it says ORDER_STATUS_COMPLETE then I will understand. 
Also the method is called 'order.archive' but all it seems to be doing is saving the record, not archiving it